app/views/admin_posts/_admin_post_form.html.erb

```diff
@@ -67,6 +67,12 @@
             <p class="footnote"><%= t(".translated_post.footnote_tags") %></p>
           <% end %>
         </dd>
+        <dt class="moderated comments">
+          <%= f.check_box :moderated_commenting_enabled %>
+        </dt>
+        <dd class="moderated comments">
+          <%= f.label :moderated_commenting_enabled, t("comments.commentable.permissions.moderated_commenting.enable") %>
+        </dd>
```

Comment:
Is this supposed to be reversed from how other inputs are displayed?
![new Admin Post screen showing moderation checkbox on the left and label on the right as well as other existing fields](https://github.com/otwcode/otwarchive/assets/13002992/da16cd8c-1e58-4158-886f-61f0af3d6beb)

Reply:
Yup, that's how it is on the work form as well:
<img width="830" alt="Privacy section of the work form. Checkboxes for 'Only show your work to registered users' and 'Enable comment moderation' are on the left." src="https://github.com/otwcode/otwarchive/assets/907055/a2abaa1d-f73f-4442-aabd-958e1ad28d66">

Reply:
Huh, my brain apparently is fine with multiple but gets weirded out with only 1 checkbox that way... not sure I like having fields in both columns in general, but that's not in scope for this PR